mobile/src/board/BoardTypes.ts

```diff
@@ -9,6 +9,11 @@ export interface BoardListItem {
   remainingSeats: number;
 }
 
+export interface UserAction {
+  kind: string;
+  payload: any;
```

Comment:
would be nice if your payload type could be more precise
something like RotationPayload | PlacementPayload | MovePayload